src/screens/Home/Home.tsx

```diff
@@ -60,11 +60,11 @@ function Home() {
     <>
       <Header />
       <section className="flex flex-col items-center mb-10">
-        <div className="flex justify-center w-full max-w-3xl">
-          <HeroIcon />
-        </div>
-        <p className="w-1/2 mt-8 font-sans font-thin text-center text-gray-700">
-          ppoker help teams during their planning sessions
+        <p className="w-1/2 mt-8 font-sans font-thin text-center text-gray-700 text-4xl">
```

Comment:
I think we can make this full for mobile support instead of `w-1/2`

```suggestion
        <p className="w-full md:w-1/2 mt-8 font-sans font-thin text-center text-gray-700 text-4xl">
```

Same with the next paragraph